src/App.js

```diff
@@ -1,13 +1,24 @@
 import React from "react";
-
 import Bookings from "./Bookings";
 import "./App.css";
+import fakeCities from "./data/fakeCities.json";
+import TouristInfoCards from "./TouristInfoCards";
+import Heading from "./Heading";
+import Footer from "./Footer";
 
 const App = () => {
   return (
     <div className="App">
-      <header className="App-header">CYF Hotel</header>
+      <Heading />
+      <TouristInfoCards cities={fakeCities} />
       <Bookings />
+      <Footer
+        arr={[
+          "123 Fake Street, London, E1 4UD",
+          "hello@fakehotel.com",
+          "0123 456789"
+        ]}
+      />
```

Comment:
I'd pass each of this as a separate prop rather than as an array together - e.g.
```
<Footer address-{'123 FakeStreet'} email={'hello@fakemail'} phoneNumber={'0123 45678' />}
```
just because they're all strings than represent something specific rather than all the same thing